nall/nall/priority-queue.hpp

```diff
@@ -26,10 +26,12 @@ struct priority_queue<T[Size]> {
 
   template<typename F>
   auto step(u32 clocks, const F& callback) -> void {
-    clock += clocks;
-    while(size && ge(clock, heap[0].clock)) {
+    u32 baseClock = clock;
```

Comment:
Nitpick:
```suggestion
    u32 nextClock = clock + clocks;
```
would be a little more concise in usage below.